tests/conftest.py

```diff
@@ -1896,12 +1897,29 @@ async def dont_remove_caplog_handlers(request, monkeypatch):
     original_apply_config = inmanta_logging.FullLoggingConfig.apply_config
 
     def patched_apply_config(self) -> None:
-        caplog_handlers = [h for h in logging.root.handlers if is_caplog_handler(h)]
+        # Make sure the root log level is not altered.
```

Comment:
Is this always desired? What with test cases that set an explicit log level?Could we pick the higher (more verbose) of the two?